crates/astria-sequencer/src/transaction/mod.rs

```diff
@@ -157,6 +173,21 @@ pub(crate) async fn execute<S: StateWriteExt>(
         .await
 }
 
+#[derive(Debug)]
+pub(crate) struct InvalidChainId(pub(crate) String);
```

Comment:
why `pub(crate)`?

Reply:
Admittedly I just matched the InvalidNonce error publicity since these should be consumed similarly.